src/unistd.rs

```diff
@@ -669,8 +669,13 @@ pub fn getcwd() -> Result<PathBuf> {
                 }
             }
 
+            #[cfg(not(target_os = "hurd"))]
+            const PATH_MAX: usize = libc::PATH_MAX as usize;
+            #[cfg(target_os = "hurd")]
+            const PATH_MAX: usize = 1024;
```

Comment:
Same here, might be better to have a comment explaining the reason for this magic number

Reply:
ok, ditto